client/app/expenses/hooks/useUpdatePaymentStatus.ts

```diff
@@ -0,0 +1,31 @@
+import { useMutation } from '@tanstack/react-query'
+import { useQueryClient } from '@tanstack/react-query'
+import { toast } from '@/_hooks/useToast'
+import { isErrorWithMessage } from '@/_utils'
+import { updatePaymentStatusApi } from '../api'
+
+export const useUpdatePaymentStatusMutation = () => {
+  const queryClient = useQueryClient()
+
+  const { mutate, isPending } = useMutation({
+    mutationFn: updatePaymentStatusApi,
+    onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: ['expenses'] })
```

Comment:
You need to update TODO for the main page!

<img width="1068" alt="Screenshot 2024-04-09 at 3 38 50 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/68d8f5d6-1ee0-471f-a07f-48fde17f493b">

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/113/commits/1472fee8c826a4f663622bd443c7a5218a0d83d6